randovania/data/json_data/prime1.txt

```diff
@@ -1224,7 +1227,7 @@ Asset id: 1282373491
       Any of the following:
           Slope Jump (Advanced)
           All of the following:
-              Space Jump Boots
+              Space Jump Boots and No Gravity Suit and Enabled Allow Underwater Movement without Gravity Suit
```

Comment:
The only path without tricks enabled has a `No Gravity Suit`. Is that correct?

Reply:
that looks like a mistake I made, should just say space jump